ext/nnstreamer/tensor_decoder/tensordec-flexbuf.cc

```diff
@@ -33,6 +33,26 @@
  *                          Blob | <tensor data>
  *                         }
  * }
+ *
+ * If you want to convert tensors to your own binary format of the flexbuffers,
+ * You can use custom mode of the tensor decoder.
```

Comment:
Could I ask difference between custom easy and custom? 

if tensorfilter has custom-easy and custom, but custom-easy maps to custom in other types of element, I wonder wouldn't it create any confusion 

| element        | custom-hard                            | custom-easy |
|----------------|----------------------------------------|-------------------|
| tensor-filter  | custom                                 | custom-easy       |
| tensor-decoder | (is this sure if it won't be created?) | custom            |
| tensor-??? | ??? | ???           |

Reply:
I see, i guess this can be handled in different PR.